### DISCUSSION TITLE
Place to share your scripts?

### DISCUSSION THREAD

**Discussion by sj3fk3** [custom-components/pyscript#198] [Ideas]
It would be nice if we had a common place where we could post or showcase our pyscripts. I want to get started on a watchdog app that monitors sensors, because too often a automations don't trigger when sensors are silently failing (z-wave/zigbee/battery/issues) and instead of reinventing the wheel... I would be fine stealing someones else's bright idea ;-)

**Comment by dlashua** [comment]
People are using a combination of things for this now. Some in the Home Assistant Forums, others in the Wiki, and here, in Discussions.

I've got a pyscript similar to this, but it's very specific to my environment and hasn't been adjusted for sharing/configuration. However, it's pretty simple in execution, so I've never considered sharing it. If I clean it up for sharing, I'll let you know.

**Reply by dorbitbrown** [reply]
I'm intrigued, but nothing stands out among the Google search results for "home assistant wiki" or "pyscript wiki".  Which site did you have in mind?
